fenicsadapter/fenicsadapter.py

```diff
@@ -133,43 +115,17 @@ def update_coupling_expression(self, coupling_expression, data):
         """
         coupling_expression.update_boundary_data(data, self._coupling_mesh_vertices[:, 0], self._coupling_mesh_vertices[:, 1])
 
-    def create_point_sources(self, fixed_boundary, data=None):
+    def create_point_sources(self, fixed_boundary):
```

Comment:
This one-liner function looks a bit unnecessary. We should probably rename it. I don't think that we should add the `fixed_boundary` as an argument to `update_point_sources`, since this will force the user to provide the `fixed_boundary` over and over again even though it is always the same boundary (and it also **has to be** the same boundary).

Reply:
`create_point_sources` can be removed and `fixed_boundary` can be included in `initialize()` as an optional argument.